kep/9-capacity-scheduling-for-ml-dl-workloads-based-on-scheduler-framework/README.md

```diff
@@ -132,47 +141,38 @@ bogged down.
-
-Initially, UserA consumes 2 GPUs and UserB consumes 3 GPUs. Both consumptions are within the reserved "guaranteed" quota range.
-<div style="text-align: center">
-<img src="./1.png"/>
-</div>
+We assume two elastic quotas are defined: QuotaA (min:4, max:6) and QuotaB (min:6, max:8), the quota unit is the number of GPUs. UserA in green color consumes QuotaA and UserB in red color consumes QuotaB. The entire cluster has 10 GPUs available hence the sum of quota min is equal to the cluster capacity.
 
+Initially, UserA consumes 2 GPUs and UserB consumes 3 GPUs. Both consumptions are within the reserved "min" quota range.
+![1](./1.png)
 
-Next, UserA consumes 4 GPUs reaching the "guaranteed" quota of QuotaA. UserB still consumes 3 GPUs. The cluster still has sufficient GPU resources hence no resource stealing happens.
-<div style="text-align: center">
-<img src="./2.png"/>
-</div>
 
+Next, UserA consumes 4 GPUs reaching the "min" quota of QuotaA. UserB still consumes 3 GPUs. The cluster still has sufficient GPU resources hence no resource stealing happens.
+![2](./2.png)
 
-Assuming UserA attempts to use the rest of 3 unused GPUs in the cluster,  he can only use 2 more GPUs since his quota "hard" is 6 GPUs. Note that these 2 GPUs are "borrowed" from UserB's "Guaranteed" quota since UserB does not use up it. 
-<div style="text-align: center">
-<img src="./3.png"/>
-</div>
 
+Assuming UserA attempts to use the rest of 3 unused GPUs in the cluster,  he can only use 2 more GPUs since his quota "max" is 6 GPUs. Note that these 2 GPUs are "borrowed" from UserB's "Guaranteed" quota since UserB does not use up it. 
+![3](./3.png)
 
-Later, if UserB demands 8 GPUs, 5 more compared to his current consumption, his request cannot be immediately satisfied since only one unused GPU left in the cluster. Since UserB's "guaranteed" quota is 6 GPUs, the preemption algorithm ensures that three GPUs will be returned from UserA.  However, no more GPUs can be assigned to UserB since UserA's "guaranteed" quota has to be honored as well. In the end, UserA consumes 4 GPUs and UserB consumes 6 GPUs, satisfying each user's "guaranteed" quota.
-<div style="text-align: center">
-<img src="./4.png"/>
-</div>
+Later, if UserB demands 8 GPUs, 5 more compared to his current consumption, his request cannot be immediately satisfied since only one unused GPU left in the cluster. Since UserB's "min" quota is 6 GPUs, the preemption algorithm ensures that three GPUs will be returned from UserA.  However, no more GPUs can be assigned to UserB since UserA's "min" quota has to be honored as well. In the end, UserA consumes 4 GPUs and UserB consumes 6 GPUs, satisfying each user's "min" quota.
```

Comment:
In this case, it doesn't help if UserB get another 3GPU (less than its requested 8 GPUs), isn't it?
Another question is: how to prioritize these two quotas upon preemption? By Pod's priority?

Reply:
> In this case, it doesn't help if UserB get another 3GPU (less than its requested 8 GPUs), isn't it?
-----
I'm not sure I understand this question： the requested doesn't represent pod.spec.resources.requested. It means the sum of the resource requests for many jobs. 3GPU means the UserB get some resource to run the part of jobs.

Reply:
> how to prioritize these two quotas upon preemption? By Pod's priority?
---

The current design does not differentiate between different namespaces. Just Inter-Quota or Intra-Quota. If if Preemptor.Request + Quota.allocated > Quota.min, the preempt will happen Inter-Quota. All the pods belongs to other Quotas that allocates more resources than its min in a node will be selected as the potential victims. The subsequent selection strategy remains the current logic: priority, PDB, createtime.

Reply:
This comment is still outstanding.

Reply:
Done